### PR TITLE
Added in the Newcomer's guide to not to click on the update branch bu…

### PR DESCRIPTION
**Pull request by Nanu21**
…tton after submitting the pull request

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .idea/coala.iml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      <excludeFolder url="file://$MODULE_DIR$/venv" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="PyDocumentationSettings">
+    <option name="format" value="PLAIN" />
+    <option name="myDocStringFormat" value="Plain" />
+  </component>
+</module>
```
Comment (abhishalya) [inline .idea/coala.iml:14]: These files are not relevant to us. Please exclude these from your commit.

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -442,7 +442,7 @@ Step 5. Creating a Pull Request
 
 Now that your commit has been sent to your fork, it is time
 to create a ``Pull Request``. You can do this by accessing your fork on GitHub
-and clicking ``New Pull Request``.
+and clicking ``New Pull Request``. After you submit your pull request, **DO NOT click the 'Update Branch' button**. When asked for a rebase, consult <http://coala.io/rebase> instead.
```
Comment (abhishalya) [inline docs/Developers/Newcomers_Guide.rst:445]: This line is a bit too long, please break it down into multiple lines.

**Review by abhishalya — CHANGES_REQUESTED** [review]

File: docs/Developers/Newcomers_Guide.rst (hunk 2 above)
Comment (utkarsh2102) [reply docs/Developers/Newcomers_Guide.rst:445]: I wonder if it's even needed in the first place!?

**Comment by abhishalya** [comment]
Closing as the PR was abandoned.

**Pull request closed without merging**